schemas/purge-cache-request-list.yml

```diff
@@ -0,0 +1,48 @@
+$schema:  http://json-schema.org/draft-04/schema#
+title:        "Open Purge Request List"
+description: |
+  A list of currently open purge-cache requests.
+type:         object
+properties:
+  continuationToken:
+    type: string
+    description: Passed back from Azure to allow us to page through long result sets.
```

Comment:
That's an option for the list-all endpoint that I want to be around for admin purposes. I can make two schemas if you'd like instead, but they're 99% similar.